app/components/pages/Dashboard/Dashboard.js

```diff
@@ -1,90 +1,78 @@
+const RightAlignedButton = styled(Box)`
+  margin: ${th('space.3')} 0 ${th('space.1')};
+  text-align: right;
   display: none;
   ${media.mobileUp`display: block`};
 `
 
+const DashboardPanel = styled(Tabs.Panel)`
+  min-height: 300px;
+`
+
 const MobileOnlyStickyFooter = styled(StickyFooter)`
   display: block;
   ${media.mobileUp`display: none;`};
 `
 
-const Header = styled.th`
-  text-align: left;
-  padding: 3px 6px;
+const EmptyListMessage = styled.div`
+  margin-top: ${th('space.7')};
+  text-align: center;
+  color: ${th('colorTextSecondary')};
 `
 
-const Cell = styled.td`
-  border-top: 1px solid grey;
-  padding: 3px 6px;
+const CenterdSmallParagraph = styled(SmallParagraph)`
+  color: ${th('colorTextSecondary')};
+  text-align: center;
 `
 
-const Dashboard = ({ manuscripts, deleteManuscript, createManuscript }) => (
-  <React.Fragment>
-    <FormH2>Dashboard Dummy Page</FormH2>
-
-    <BoxHiddenFromMobile mb={4}>
-      <Button
-        data-test-id="desktop-new-submission"
-        onClick={createManuscript}
-        primary
-      >
-        New Submission
-      </Button>
-    </BoxHiddenFromMobile>
-
-    <FormH3>All manuscripts</FormH3>
-
-    {!manuscripts.length && <p>No manuscripts to display</p>}
-
-    {!!manuscripts.length && (
-      <table data-test-id="manuscripts">
-        <thead>
-          <tr>
-            <Header>Title</Header>
-            <Header>Stage</Header>
-            <Header />
-          </tr>
-        </thead>
-        <tbody>
-          {manuscripts.map(manuscript => (
-            <tr key={manuscript.id}>
-              <Cell data-test-id="title">
-                <Paragraph data-test-id="title">
-                  <Link
-                    to={`${manuscript.formState ||
-                      `/submit/${manuscript.id}/author`}`}
-                  >
-                    {manuscript.meta.title || '(Untitled)'}
-                  </Link>
-                </Paragraph>
-              </Cell>
-              <Cell data-test-id="stage">
-                <ManuscriptStatus statusCode={manuscript.clientStatus} />
-              </Cell>
-              <Cell>
-                <Button onClick={() => deleteManuscript(manuscript.id)} small>
-                  Delete
-                </Button>
-              </Cell>
-            </tr>
-          ))}
-        </tbody>
-      </table>
-    )}
+const Dashboard = ({ createManuscript }) => (
+  <Flex>
+    <Box flex="1 1 auto" mx={[0, 0, 0, '16.666%']}>
+      <RightAlignedButton>
+        <Button
+          data-test-id="desktop-new-submission"
+          onClick={createManuscript}
+          primary
+        >
+          New Submission
+        </Button>
+      </RightAlignedButton>
+      <Tabs>
+        <Tabs.List>
+          <Tabs.Tab>Submissions</Tabs.Tab>
+          <Tabs.Tab>Archive</Tabs.Tab>
+        </Tabs.List>
+        <DashboardPanel data-test-id="manuscripts">
+          <ManuscriptsQuery query={ALL_MANUSCRIPTS}>
```

Comment:
I like this change. Having ManuscriptsQuery & operations in separate files makes it much more readable, imo.

But this is a small departure from our prior decision to use the top-level page component (`index.js`) to handle all graphql queries and mutations. Without fully understanding why that decision was made, I would probably highlight this with @tamlyn when he gets back.

It might be worth discussing appropriate patterns for organising graphql on the frontend? LGTM

Reply:
I've extracted the content of these dashboard panels into their own files as each one should be it's own 'top' level file of sorts, which would kind of fit with what had been agreed previously. I have a feeling these panels may be mapped to routes in the future too so doing this would make it easier to alter later on.